src/workflows/bidsConcatBetaTmaps.m

```diff
@@ -15,8 +15,14 @@ function bidsConcatBetaTmaps(opt, funcFWHM, deleteIndBeta, deleteIndTmaps)
   % :param deleteIndTmaps: decide to delete t-maps
   % :type funcFWHM: (boolean)
   %
+  % When concatenating betamaps:
+  %
+  % Ensures that there is only 1 image per "contrast".
+  % Creates a tsv that lists the content of the 4D image.
+  % This TSV is in the subject level GLM folder where the beta map came from.
+  % This TSV file is named ``sub-subLabel_task-taskName_space-space_labelfold.tsv``.
+  %
```

Comment:
% in tsv file, `folds` represent the runs/repetitions of condition, and `labels` represent trial type (condition).

Reply:
Yeah I was not sure what terms to use because people in different fields use different words to refer to the same thing...

Should we stick with BIDS nomenclature: `run` and `trial_type` ?

Reply:
I like BIDS labelling. and it is true that folds are used in some other toolboxes... only thing about the `run` is, it might be confusing when one has 4D-maps with repetitions (run 1 repetition1, run1 repetition2, ...). No?

Reply:
ha yes... you are right...

but actually, it could be useful to have both run and repetition, no?

the most "verbose" way of doing this is taking the whole name of the regressor for each image from SPM.mat

this should contain all the info you need: and it is then up to the user to parse this string.

so having the `run` info AND the `regressor_name`?  there will be a bit of information redundancy because the run number is in the regressor name but...meh...

Reply:
at the current form, if i am not mistaken, we needed to change the trial type (in _events.tsv file) for GLM to have separate repetitions modelled as separate regressors (betas). so technically it would work like this: 

repetition | runs | trial_type
--|-- | --
1 | 1 | Correct_Task*bf(1)_rep1
2 | 1 | Correct_Task*bf(1)_rep2
1 | 2 | Correct_Task*bf(1)_rep1
2 | 2 | Correct_Task*bf(1)_rep2
1 | 1 | Incorrect_Task*bf(1)_rep1
2 | 1 | Incorrect_Task*bf(1)_rep2
1 | 2 | Incorrect_Task*bf(1)_rep1
2 | 2 | Incorrect_Task*bf(1)_rep2

Reply:
yes sort of (I think that SPM always appends the BF at the end) 

repetition | runs | trial_type
--|-- | --
1 | 1 | Correct_Task_rep1*bf(1)
2 | 1 | Correct_Task_rep2*bf(1)
1 | 2 | Correct_Task_rep1*bf(1)
2 | 2 | Correct_Task_rep2*bf(1)
...

But the issue that not everyone will have repetitions, but everyone will have `runs` and "`trial_type`".

Also the names of the regressors are longer:

```
'Sn(1) listening*bf(1)'
```

Where `Sn(X)` is the number of the 'session' in SPM lingo (more or less a 'run' in the BIDS world... why use the same terms, huh?)